config/clusters/2i2c/neurohackademy.values.yaml

```diff
@@ -0,0 +1,108 @@
+dex:
+  enabled: true
+  hubHostName: neurohackademy.2i2c.cloud
+
+staticWebsite:
+  enabled: true
+  source:
+    git:
+      repo: https://github.com/neurohackademy/zoom2022
+      branch: master
+  ingress:
+    host: neurohackademy.2i2c.cloud
+    path: /zoom
+  githubAuth:
+    enabled: true
+
+jupyterhub:
+  ingress:
+    hosts:
+      - neurohackademy.2i2c.cloud
+    tls:
+      - secretName: https-auto-tls
+        hosts:
+          - neurohackademy.2i2c.cloud
+  custom:
+    2i2c:
+      add_staff_user_ids_to_admin_users: true
+      add_staff_user_ids_of_type: "github"
+    homepage:
+      templateVars:
+        org:
+          name: NeuroHackademy
+          url: https://neurohackademy.org/
+          logo_url: https://user-images.githubusercontent.com/118582/178122521-aa5b83e7-f898-415a-89ff-dc288f599b4f.png
+        designed_by:
+          name: 2i2c
+          url: https://2i2c.org
+        operated_by:
+          name: 2i2c
+          url: https://2i2c.org
+        funded_by:
+          name: The National Institutes of Health grant 2R25MH112480-06
+          url: https://reporter.nih.gov/search/ydTvTwXxk0yd6eGdRznbLQ/project-details/10409452
+  singleuser:
+    # User image: https://quay.io/repository/arokem/nh-jhub-2022?tab=tags
+    image:
+      name: quay.io/arokem/nh-jhub-2022
+      tag: "3d441bdb82f6"
+    # Uncomment this section after resolution of PR https://github.com/2i2c-org/infrastructure/pull/2758
+    # nodeSelector:
```

Comment:
https://github.com/2i2c-org/infrastructure/pull/2758#issuecomment-1622684841 has this sorted, so this needs the nodeSelector *and* some tolerations on this. See https://infrastructure.2i2c.org/howto/features/dedicated-nodepool/

Reply:
Done in [`d6dc270` (#2762)](https://github.com/2i2c-org/infrastructure/pull/2762/commits/d6dc270359ff2e66834d0d51f3f7fc8cd1eeca1c)